consensus/core/src/metrics.rs

```diff
@@ -237,9 +267,15 @@ impl NodeMetrics {
                 "Designates whether the synchronizer scheduler task to fetch blocks is currently running",
                 registry,
             ).unwrap(),
-            fetched_blocks: register_int_counter_vec_with_registry!(
-                "fetched_blocks",
-                "Number of fetched blocks per peer authority via the synchronizer.",
+            synchronizer_fetched_blocks_by_peer: register_int_counter_vec_with_registry!(
+                "synchronizer_fetched_blocks_by_peer",
+                "Number of fetched blocks per peer authority via the synchronizer and also by block authority",
+                &["peer", "type"],
```

Comment:
The cardinality of # peers * # authorities could be just too large (>10000). We can test in private testnet but for now I'm splitting them into two metrics.